Reply/app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml

```diff
@@ -13,6 +13,5 @@
   the License.
   -->
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
-    <background android:drawable="@color/reply_blue_700"/>
```

Comment:
I would re-add the colors files and hardcode baseline hex values for `tertiary container` and `on tertiary container` so you can use those for the svg's background and for the `tint` of the foreground.

You might also want to use Android Studio's "[Image Asset Studio](https://developer.android.com/studio/write/image-asset-studio#access)" to replace the app icon pngs under `res/mipmap` with the new background and foreground colors